nix/options.nix

```
@@ -30,6 +30,16 @@ in
       '';
     };
 
+    deployment.targetUser = mkOption {
+      type = types.nullOr types.str;
+      default = "root";
+      description = ''
+        This option specifies the username to be used by
+        NixOps on the remote system to execute deployment
+        operations.
```

"This option specifies..." is redundant in context. Also, let's specify a bit more of the details.

```suggestion
        The username to be used by NixOps by SSH when connecting to the
        remote system.
        
        If this user is not <literal>root</literal>, it must be able to become root using the command in deployment.privilegeEscalationCommand .
        When null, NixOps uses the current user's name. 
```